src/cdk/layout/breakpoints-observer.ts

```diff
@@ -58,10 +58,14 @@ export class BreakpointObserver implements OnDestroy {
     let queries = coerceArray(value);
     let observables = queries.map(query => this._registerQuery(query).observable);
 
-    return combineLatest(observables, (a: BreakpointState, b: BreakpointState) => {
-      return {
-        matches: !!((a && a.matches) || (b && b.matches)),
-      };
+    // Regular function expression is used as the arguements object cannot be used in
+    // arrow functions.
+    return combineLatest<BreakpointState>(observables, function() {
+      const output: BreakpointState = { matches: false };
+      Array.from(arguments).forEach((breakpointState: BreakpointState) => {
```

Comment:
You could use a regular loop here to avoid having to convert the arguments to an array and to be able to break out of it if you hit a match.